samples/firefox-accounts/build.gradle

```diff
@@ -36,7 +36,12 @@ android {
 }
 
 dependencies {
-    implementation project(':service-firefox-accounts')   
+    implementation project(':service-firefox-accounts')
+
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation ('me.dm7.barcodescanner:zxing:1.9') {
```

Comment:
This will probably have a follow up PR to start moving this out of `samples` somehow

Reply:
Do we know how big the library is (including its dependencies)? There used to be methodcount.com which was able to provide that number easily - but it seems to be down. :(

If it's quite big then I wonder if we should make this an supplemental "add-on" library?